### ISSUE TITLE
Crash during campaign

### ISSUE THREAD

**Issue by murphy-edwards** [tonihele/OpenKeeper#314]
About one minute into the first level of the campaign, the game crashes with this error message in a dialog box:
"Uncaught exception thrown in Thread[main,5,main]
MissingResourceException: Can't find bundle for base name Interface/Texts/Text, locale en_US"

edit: here's the full stack trace:
```
Oct 04, 2018 8:37:59 PM com.jme3.app.LegacyApplication handleError
SEVERE: Uncaught exception thrown in Thread[main,5,main]
java.util.MissingResourceException: Can't find bundle for base name Interface/Texts/Text, locale en_US
	at java.util.ResourceBundle.throwMissingResourceException(ResourceBundle.java:1573)
	at java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1396)
	at java.util.ResourceBundle.getBundle(ResourceBundle.java:824)
	at toniarts.openkeeper.Main.getResourceBundle(Main.java:556)
	at toniarts.openkeeper.world.object.ObjectControl.<init>(ObjectControl.java:70)
	at toniarts.openkeeper.world.object.GoldObjectControl.<init>(GoldObjectControl.java:50)
	at toniarts.openkeeper.world.object.ObjectLoader.getControl(ObjectLoader.java:139)
	at toniarts.openkeeper.world.object.ObjectLoader.load(ObjectLoader.java:121)
	at toniarts.openkeeper.world.object.ObjectLoader.load(ObjectLoader.java:96)
	at toniarts.openkeeper.world.object.ObjectLoader.load(ObjectLoader.java:81)
	at toniarts.openkeeper.world.ThingLoader.addRoomGold(ThingLoader.java:326)
	at toniarts.openkeeper.world.room.control.RoomGoldControl.putGold(RoomGoldControl.java:81)
	at toniarts.openkeeper.world.room.control.RoomGoldControl.addItem(RoomGoldControl.java:55)
	at toniarts.openkeeper.world.WorldState.addGold(WorldState.java:1304)
	at toniarts.openkeeper.game.task.worker.CarryGoldToTreasuryTask.executeTask(CarryGoldToTreasuryTask.java:66)
	at toniarts.openkeeper.world.creature.CreatureControl.onAnimationCycleDone(CreatureControl.java:450)
	at toniarts.openkeeper.world.creature.CreatureControl.playStateAnimation(CreatureControl.java:469)
	at toniarts.openkeeper.world.creature.CreatureControl.onAnimationStop(CreatureControl.java:435)
	at toniarts.openkeeper.world.animation.AnimationLoader$3.onAnimCycleDone(AnimationLoader.java:168)
	at com.jme3.animation.AnimControl.notifyAnimCycleDone(AnimControl.java:301)
	at com.jme3.animation.AnimChannel.update(AnimChannel.java:358)
	at com.jme3.animation.AnimControl.controlUpdate(AnimControl.java:362)
	at com.jme3.scene.control.AbstractControl.update(AbstractControl.java:128)
	at com.jme3.scene.Spatial.runControlUpdate(Spatial.java:736)
	at com.jme3.scene.Spatial.updateLogicalState(Spatial.java:879)
	at com.jme3.scene.Node.updateLogicalState(Node.java:230)
	at com.jme3.scene.Node.updateLogicalState(Node.java:241)
	at com.jme3.app.SimpleApplication.update(SimpleApplication.java:242)
	at com.jme3.system.lwjgl.LwjglWindow.runLoop(LwjglWindow.java:506)
	at com.jme3.system.lwjgl.LwjglWindow.run(LwjglWindow.java:589)
	at com.jme3.system.lwjgl.LwjglWindow.create(LwjglWindow.java:427)
	at com.jme3.app.LegacyApplication.start(LegacyApplication.java:463)
	at com.jme3.app.LegacyApplication.start(LegacyApplication.java:424)
	at com.jme3.app.SimpleApplication.start(SimpleApplication.java:125)
	at toniarts.openkeeper.Main.main(Main.java:117)
```

**Comment by tonihele** [comment]
And you do have \assets\Converted\Interface\Texts\Text.properties file existing in the system? And if Linux/Mac, is also the case sensitivity correct?

**Comment by murphy-edwards** [comment]
I am using the GOG Version of the game. It's in English. I can't tell from the assets what locale it is, though.

I'm running this on Ubuntu Linux 18.04, and that file does exist and is correctly named.

**Comment by tonihele** [comment]
Hmm, can you reproduce this 100% of the time? By the looks of it, this should happen immediately when you open a level with already gold in a treasury/dungeon heart. This means almost every level. Could you verify this?

**Comment by tonihele** [comment]
I don't really know what is going on, but I reduced the number of resource bundle references. ResourceBundles are cached. But getting them is a bit of an extra hassle and certainly unneeded in class instances. I replaced the class instances with a static one we actually already had existing.
Given that you shouldn't be even able to start the game(?) if the main text bundle is somehow failing, and now that we never try to acquire it again.... This problem should be solved. I'm waiting for your confirmation on this. As I was not even able to reproduce this ever.
